Update 10-supp-addressing-data.Rmd

eigensong opened the pull request:
Dear Carpentries Community,

In my instructor training I learnt about "fluid representations" where multiple words for the same concept can confuse learners. I noticed that in the Addressing Data most of the lesson is really careful to avoid fluid representations, by consistently using the word addressing to describe accessing parts of a data frame, rather than mixing in words like subsetting or indexing. However, the titles for the three sections of the types of addressing are:
Addressing by Index
Addressing by Name
Logical Indexing

To fix this issue I suggest replace 'indexing' with 'addressing' across the whole document (as that terminology is only used right at the end) and changing the title 'Logical Indexing' to 'Addressing by Logical Vector'.

Cheers,

Elena

eigensong committed:
Update 10-supp-addressing-data.Rmd

Replace 'indexing' with 'addressing' so the whole document uses consistent terminology

HaoZeke reviewed (changes requested):
Thanks for this, please consider adding a line stating that this particular form of addressing is commonly called "Logical Indexing".

eigensong committed:
Update 10-supp-addressing-data.Rmd

Add a sentence explaining that the common term for "addressing by a logical vector" is "logical indexing".

eigensong commented:
I've added a sentence. Is that what you had in mind?

HaoZeke reviewed (approved):
Fantastic work. Thanks.

HaoZeke commented:
Ah I just realized something. This is a PR which is aimed at your own `master`, please point one at the upstream [repo here](https://github.com/swcarpentry/r-novice-inflammation/compare).

eigensong commented:
Right, I see. Is there a way to convert this pull request to one for the upstream repo? Or do I need to start a new pull request on that repo? Sorry, this is a learning exercise for me in using github!

HaoZeke commented:
No worries, [apparently it is possible](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/changing-the-base-branch-of-a-pull-request) for you to edit this one to point to the SWC repo.

eigensong commented:
Following those instructions, I get a drop down menu, but I can't see how to select the swc version from that list.

![image](https://user-images.githubusercontent.com/42173162/83089023-9d42c400-a0d8-11ea-8f26-99d2d1151a64.png)

HaoZeke commented:
o.O Ah I guess it isn't possible then >.<
Please open a new one then?

eigensong commented:
OK, thanks for getting back to me. Do I just close this one?

HaoZeke commented:
Yup that would be best
